### PR TITLE
Scaffold Application Structure

### LINKED ISSUE #3: Application Setup

### PR DESCRIPTION
**Pull request by as-cii**
Close #3.

I have scaffolded the application structure with Yeoman Ember. @exalted if you agree, let's stick with this and change it later if something does not fit us.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# editorconfig.org
+
+root = true
+
+
+[*]
+
+# Change these settings to your own preference
+indent_style = space
+indent_size = 4
```
Comment (exalted) [inline .editorconfig]: 2

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1 @@
+* text=auto
```
Comment (exalted) [inline .gitattributes]: Is it necessary? Also, missing new line at the EOF

File: app/scripts/routes/application_route.coffee [hunk 3]
```diff
@@ -0,0 +1,4 @@
+CaptrWeb.ApplicationRoute = Ember.Route.extend
+    # admittedly, this should be in IndexRoute and not in the
+    # top level ApplicationRoute; we're in transition... :-)
```
Comment (exalted) [inline app/scripts/routes/application_route.coffee]: @as-cii is this you? :smile:

File: scripts/bootstrap [hunk 4]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+echo "Preparing the awesome..."
+
+echo -n "Installing required node packages... "
+npm install &> /dev/null
```
Comment (exalted) [inline scripts/bootstrap:6]: How about `>` only instead?

File: scripts/bootstrap [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+echo "Done."
+
+echo -n "Installing javascript dependencies... "
+bower install --silent
+cd test/
+bower install --silent
```
Comment (exalted) [inline scripts/bootstrap:15]: Installing twice?

Comment (as-cii) [inline scripts/bootstrap:15]: Nope, it installs bower packages for the application and for the tests.

File: scripts/bootstrap (hunk 4 above)
Comment (as-cii) [inline scripts/bootstrap:6]: Nope, it will be too verbose. I could pass `--silent` but some messages would be displayed anyway. Let's keep this for now?

Comment (exalted) [inline scripts/bootstrap:6]: :+1:

File: scripts/bootstrap (hunk 5 above)
Comment (exalted) [inline scripts/bootstrap:15]: Ah. That stinks! :disappointed:

**Pull request merged**